src/test/ui/missing-trait-bounds/missing-trait-bounds-for-method-call.stderr

```diff
@@ -7,11 +7,22 @@ LL | struct Foo<T> {
 LL |         self.foo();
    |              ^^^ method cannot be called on `&Foo<T>` due to unsatisfied trait bounds
    |
-   = note: the following trait bounds were not satisfied:
-           `T: Default`
-           which is required by `Foo<T>: Bar`
-           `T: Bar`
-           which is required by `Foo<T>: Bar`
+note: trait bound `T: Bar` was not satisfied because of the requirements of the implementation of `Bar` for `_`
```

Comment:
The "duplicated" output is because it points at *both* `Default` and `Bar` bounds as unsatisfied for `Bar`, while it never talks about `T: !Bar` beyond the first mention.